src/features/Status/StatusHeader.tsx

```diff
@@ -111,7 +111,11 @@ const StatusHeader: React.SFC<IStatusHeaderProps> = ({
       break;
     case DetailedState.CONFIRMED:
       text = CONSTANTS.CONFIRMED_STATUS_TEXT;
-      buttons = [hackPassButton, travelButton, withdrawButton];
+      if (!settings.isRemote) {
```

Comment:
Nit: consider flipping this to `settings.isRemote`, and swapping the bodies of the if-else

Reply:
Yeah I tested applying both ways, and the data in the database seems to match up and its submitted without errors.